### PR TITLE
DAOS-9325 mercury: Update mercury to pick a fix for high cpu usage

### PR DESCRIPTION
**Pull request by frostedcmos**
- Update mercury to pick fixes for DAOS-9325 high cpu usage

PR-repos: mercury@PR-70

Signed-off-by: Alexander A Oganezov <alexander.a.oganezov@intel.com>

### CODE REVIEW

**Comment by frostedcmos** [comment]
Note: build.config changes are not done yet until a corresponding mercury rpm PR (https://github.com/daos-stack/mercury/pull/70) lands

**Review by dmiter — APPROVED** [review]

File: debian/control [hunk 1]
```diff
@@ -10,7 +10,7 @@ Build-Depends: debhelper (>= 10),
                libopenmpi-dev,
                libssl-dev,
                libyaml-dev,
-               libmercury-dev (= 2.1.0~rc4-2),
+               libmercury-dev (= 2.1.0~rc4-3),
```
Comment (johannlombardi) [inline debian/control:13]: I would suggest to unpin the version now and just use >=

**Review by johannlombardi — COMMENTED** [review]
The Mercury PR is landed.

**Review by brianjmurrell — COMMENTED** [review]
You have a typo in your commit pragma:

```
RR-repos: mercury@PR-70
```

**Review by johannlombardi — APPROVED** [review]

**Review by dmiter — APPROVED** [review]

**Comment by johannlombardi** [comment]
I have restarted the build.

**Comment by johannlombardi** [comment]
Please backport to release/2.0.

**Pull request merged**